python/paddle/fluid/initializer.py

```diff
@@ -707,36 +716,27 @@ class BilinearInitializer(Initializer):
 
         .. code-block:: python
 
-            import paddle.fluid as fluid
+            import numpy as np
+            import paddle
+            from paddle.regularizer import L2Decay
             import math
+            import paddle.nn as nn
+            paddle.disable_static()
             factor = 2
-            C = 2
-            B = 8
-            H = W = 32
-            w_attr = fluid.param_attr.ParamAttr(
-                learning_rate=0., 
-                regularizer=fluid.regularizer.L2Decay(0.),
-                initializer=fluid.initializer.Bilinear())
-            x = fluid.data(name="data", shape=[B, 3, H, W], 
-                                  dtype="float32")
-            conv_up = fluid.layers.conv2d_transpose(
-                input=x,
-                num_filters=C,
-                output_size=None,
-                filter_size=2 * factor - factor % 2,
+            w_attr = paddle.ParamAttr (
+                learning_rate=0.,
+                regularizer=L2Decay(0.),
+                #regularizer=fluid.regularizer.L2Decay(1.0),
+                initializer=nn.initializer.Bilinear()
+            )
+            data = np.random.uniform( -1, 1, [30, 10, 32] ).astype('float32')
+            data = paddle.to_tensor ( data )
+            conv_up = nn.ConvTranspose2d ( 4, 6, (3,3),
                 padding=int(math.ceil((factor - 1) / 2.)),
                 stride=factor,
-                groups=C,
-                param_attr=w_attr,
-                bias_attr=False)
-
-    Where, `num_filters=C` and `groups=C` means this is channel-wise transposed
```

Comment:
Do we need this explanation in new example code? I guess not because you are not using `num_filters` and `groups`, please confirm.

Reply:
I've confirmed and fixed.